### PR TITLE
Added DismissRecommendation example.

### PR DESCRIPTION
**Pull request by fiboknacky**

### CODE REVIEW

File: examples/Recommendations/DismissRecommendation.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+ * This example dismisses a given recommendation. To retrieve recommendations for text ads,
+ * run GetTextAdRecommendations.php.
+ */
+class DismissRecommendation
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    // Recommendation ID is the last alphanumeric portion of the value from
+    // ResourceNames::forRecommendation(), which has the format of
+    // `customers/<customer_id>/recommendations/<recommendation_id>`.
+    // Its example can be retrieved from GetTextAdRecommendations.php
```
Comment (jimper) [inline examples/Recommendations/DismissRecommendation.php]:
```suggestion
     // An example can be retrieved from GetTextAdRecommendations.php
```

File: examples/Recommendations/DismissRecommendation.php [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+use Google\ApiCore\ApiException;
+
+/**
+ * This example dismisses a given recommendation. To retrieve recommendations for text ads,
+ * run GetTextAdRecommendations.php.
+ */
+class DismissRecommendation
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    // Recommendation ID is the last alphanumeric portion of the value from
```
Comment (jimper) [inline examples/Recommendations/DismissRecommendation.php]:
```suggestion
    // Recommendation ID is the last alphanumeric portion of the resource name obtained from
```

**Review by jimper — CHANGES_REQUESTED** [review]
Some minor comment suggestions, otherwise LGTM

File: examples/Recommendations/DismissRecommendation.php [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+use Google\ApiCore\ApiException;
+
+/**
+ * This example dismisses a given recommendation. To retrieve recommendations for text ads,
+ * run GetTextAdRecommendations.php.
+ */
+class DismissRecommendation
+{
+    const CUSTOMER_ID = 'INSERT_CUSTOMER_ID_HERE';
+    // Recommendation ID is the last alphanumeric portion of resource name obtained from
```
Comment (jimper) [inline examples/Recommendations/DismissRecommendation.php]: "resource name" -> "the resource name"

Comment (fiboknacky) [reply examples/Recommendations/DismissRecommendation.php]: Done.

**Review by jimper — APPROVED** [review]
LGTM

**Review by jradcliff — APPROVED** [review]

**Pull request merged**